apistar/pipelines.py

```diff
@@ -41,7 +41,7 @@ def parameterize_by_argument_name(cls: type) -> bool:
 def get_class_id(cls: type,
```

Comment:
Worth noting that `cls: type` isn't correct here anymore (since it could also be a string).
Don't know if there's already an annotation that refers to "the type of a type annotation", but otherwise we should probably be using something like `Union[type, str]` here.

Also wonder if that'd allow us to remove the odd `str(cls)` below? We shouldn't need to do that.